### PR TITLE
Automated cherry pick of #68920: Enable insertId generation, update Stackdriver Logging Agent

### PR DESCRIPTION
**Pull request by qingling128**
Cherry pick of #68920 on release-1.12.

#68920: Enable insertId generation, update Stackdriver Logging Agent

### CODE REVIEW

**Comment by qingling128** [comment]
Could someone help with the `/ok-to-test` tag? @kawych @jingax10 @x13n

**Comment by x13n** [comment]
/lgtm cancel
Actually, after talking with @MaciekPytel I think we should have wildcard tolerations for per-node agents (metadata agent & fluentd).

**Comment by qingling128** [comment]
https://github.com/kubernetes/kubernetes/pull/69814 is created to cherry pick the `add tolerations` PR to `1.12` branch.

Can we unblock this one now (since they are not depending on each other)?

**Comment by x13n** [comment]
I'd wait for #69814 to merge, then merge this (to make sure this change is not patch-released without the other one). This needs not to remove fluentd tolerations.

**Comment by qingling128** [comment]
@x13n - Sounds good. I will wait for the other one to merge before merging this one.
@piosz - Mind taking a quick look?

**Comment by qingling128** [comment]
https://github.com/kubernetes/kubernetes/pull/69814 is merged.
@x13n Does this one look good to you now?

File: cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml [hunk 1]
```diff
@@ -100,13 +100,6 @@ spec:
       # END_PROMETHEUS_TO_SD
       nodeSelector:
         beta.kubernetes.io/fluentd-ds-ready: "true"
-      tolerations:
-      - key: "node.alpha.kubernetes.io/ismaster"
-        effect: "NoSchedule"
-      - operator: "Exists"
```
Comment (x13n) [inline cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml]: This removes the tolerations now, which it shouldn't. Please rebase & make sure they are not removed.

**Comment by qingling128** [comment]
Rebased. PTAL @x13n @kawych @MaciekPytel

**Comment by qingling128** [comment]
Seems the PR has both "approved" and "do-not-merge/cherry-pick-not-approved" labels, which blocks the merge.

I'm a bit confused. Any idea who I should talk to for getting rid of "do-not-merge/cherry-pick-not-approved"?
@x13n @kawych @MaciekPytel

**Comment by x13n** [comment]
@feiskyer is the patch release manager for 1.12.

**Comment by x13n** [comment]
This is up to patch release manager for 1.12 branch - @feiskyer, can you take a look?

**Comment by x13n** [comment]
This is up to patch release manager for 1.12 branch - @feiskyer, can you take a look?

**Comment by feiskyer** [comment]
Cherry pick approved now.

**Comment by x13n** [comment]
This is up to patch release manager for 1.11 branch - @feiskyer, can you take a look?

**Comment by qingling128** [comment]
Good to know. Thanks!

**Pull request merged**